modules/solid_mechanics/include/timeintegrators/CentralDifferenceDirect.h

```diff
@@ -0,0 +1,49 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ExplicitTimeIntegrator.h"
+#include "MooseError.h"
+#include "libmesh/sparse_matrix.h"
```

Comment:
can forward declare

Reply:
I don't understand the reason, could you explain?

Reply:
https://stackoverflow.com/questions/9906402/should-one-use-forward-declarations-instead-of-includes-wherever-possible

Reply:
I wasn't aware of this aspect of C++, thanks

Reply:
[d8b86f1](https://github.com/idaholab/moose/pull/28175/commits/d8b86f16fa90d2992962003d08f61c78bf953921)